config.default.yaml

```diff
@@ -617,6 +618,28 @@ sector:
   # 2045: 0.21
   # 2050: 0.29
 
+  district_heating_calibrate:
```

Comment:
It feels like most of these parameters shall be moved now or in the future into energy totals as they would be consistent there. In principle, most of the entries there are used for scaling the profiles. Moreover, the use of energy totals would allow country-specific scaling that can be relevant especially for large regions.
What do you think?